packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -459,7 +465,18 @@ export const Table = <
           onSort?.(newState);
           break;
         case TableActions.setFilter:
-          onFilterHandler(newState, action, previousState, instance, onFilter);
+          // Check to see if filters have changed
+          const previousFilter = previousState.filters.find(
```

Comment:
Maybe move to function to keep this part of code minimal.

Reply:
This has been moved!